Tests/SourceKitLSPTests/SemanticTokensTests.swift

```diff
@@ -833,13 +833,34 @@ final class SemanticTokensTests: XCTestCase {
         Token(line: 2, utf16index: 7, length: 8, kind: .identifier),
         Token(line: 4, utf16index: 0, length: 4, kind: .keyword),
         Token(line: 4, utf16index: 5, length: 1, kind: .identifier),
-        Token(line: 5, utf16index: 4, length: 1, kind: .identifier),
+        Token(line: 5, utf16index: 4, length: 1, kind: .function),
         Token(line: 5, utf16index: 7, length: 7, kind: .actor),
-        Token(line: 6, utf16index: 4, length: 1, kind: .identifier),
+        Token(line: 6, utf16index: 4, length: 1, kind: .function),
         Token(line: 6, utf16index: 7, length: 8, kind: .struct),
       ]
     )
   }
+
+  func testArgumentLabels() async throws {
```

Comment:
Do we have a first + second name test at all? Is the second name a parameter or an identifier?

Reply:
We do have a test for it in swift-syntax: https://github.com/apple/swift-syntax/pull/2375/files#diff-0f770f38aaec515fed644ee83824ed0dcfb2e86ec67a6da535f2de9fef15f9f3R531

Reply:
Fair enough, I'd expect one here too since the sourcekitd response could add more information if it wanted to (eg. all params + references could be "parameters").

Reply:
Added a test case